web/blog/tenzir-platform-v1.2/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Tenzir Platform v1.2
+authors: [dominiklohmann]
+date: 2024-10-23
+tags: [release, platform]
+comments: true
+---
+
+[Tenzir Platform v1.2][github-release] brings improvements to diagnostics in the
+Explorer, the ability to download charts, and many stability improvements fixes.
+
+![Tenzir Platform v1.2](tenzir-platform-v1.2.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/platform/releases/tag/v1.2.0
+
+<!-- truncate -->
+
+## Simplified Diagnostics Flow
+
+In our previous version, the diagnostics pane would automatically open when a
+pipeline encountered an error or completed without results but with at least one
+warning. While this feature aimed to provide immediate feedback, it often led to
+confusion as the pane never closed automatically. We recognized that
+automatically closing it could disrupt your workflow by causing layout shifts
+while iterating on your pipelines.
+
+With the new update, we’ve simplified the user experience: the diagnostics pane
+will no longer open or close automatically. Instead, you can easily access it by
+clicking on the diagnostics count or the newly added toasts for errors. This
+change ensures a smoother and more intuitive interaction with the diagnostics
+feature.
```

Comment:
This is really hard to understand for new users. Can we add a screenshot of the new style?

Reply:
Okay, then leave it as is. No need to overcomplicate it.